doc/source/quick_start.rst

```diff
@@ -279,14 +279,15 @@ Get file metadata
      'fileName': 'dummy_new.pdf',
      'uploadTimestamp': 1554361150000}
 
-
 Copy file
 =========
 
+Please switch to  :meth:`b2sdk.v2.Bucket.copy`.
```

Comment:
this needs a `.. info:` macro or something to stand out more, and also a rationale